CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-kms ChangeLog
 
+## 6.0.0 - 2021-04-TBD
+
+### Changed
+- **BREAKING**: Drop support for node 10.
+- Support `ed25519-signature-2020`signature suites.
+- Remove use of jsonld-signatures.
+- Remove @digitalbazaar/did-io and use bedrock-did-io
+- Update dependencies to latest:
+  [bedrock-did-io](https://github.com/digitalbazaar/bedrock-did-io/pull/2),
+  [@digitalbazaar/did-method-key](https://github.com/digitalbazaar/did-method-key-js/pull/35),
+  [webkms-switch](https://github.com/digitalbazaar/webkms-switch/pull/9).
```

Comment:
I imagined that these were going to be removed or replaced when the PRs were complete.  I think it would be OK to reference a changelog entry in this fashion.  It would also be good to reflect the version of `bedrock-did-io` for instance so that if one is going to look at a changelog, they know what version to look for.

Reply:
Updated here c230da4